src/tools/compiletest/src/header.rs

```diff
@@ -1115,6 +1115,7 @@ fn expand_variables(mut value: String, config: &Config) -> String {
     const CWD: &str = "{{cwd}}";
     const SRC_BASE: &str = "{{src-base}}";
     const BUILD_BASE: &str = "{{build-base}}";
+    const RUST_SRC_BASE: &str = "{{rust-src-base}}";
```

Comment:
Suggestion: we definitely want to update rustc-dev-guide's compiletest directive remarks to mention `{{rust-src-base}}`, which is not to be confused with `{{src-base}}`.

Reply:
Opened https://github.com/rust-lang/rustc-dev-guide/pull/2079